src/app/services/http.service.ts

```diff
@@ -12,28 +12,27 @@ export class HttpService {
 
   base_url = APPCONFIG.base_url;
 
-  httpOptions = {
-    headers: new HttpHeaders({
-       'Content-Type': 'application/json'
-    })
-  };
+  // httpOptions = {
+  //   headers: new HttpHeaders({
+  //      'Content-Type': 'application/json'
+  //   })
+  // };
```

Comment:
Please the comments

Reply:
this snippet of code is required for the other refactor PR which will be implemented in... so will keep it here for the meantime, but will be worked on in that PR